Source/Cake.CakeMail/Email/CakeMailEmailProvider.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
```

Comment:
Is this second layer of EmailProvider required?  This seems different from the way that it has been implemented in SendGrid